docs/src/CommutativeAlgebra/groebner_bases.md

```diff
@@ -10,9 +10,151 @@ using Oscar
+
+- If $h$ is nonzero, then no term of $h$ is divisible by any $\text{LM}_>(f_i)$.
+
+Without restrictions on  $>$, let elements $g\in K[x]_>$ and $f_1, \dots, f_r\in K[x]\setminus \{0\}$ be given.
+In this more general situation, Mora division with remainder allows us to compute expressions
+
+$ug = q_1f_1+\dots q_rf_r + h, \; h\in K[x]_>, \;\text{ all }\; q_i \in K[x]_>$
+
+such that:
+
+- ``u`` is a unit of $ K[x]_>$.
+- If both sides are nonzero, then $\text{LM}_>(g) \ge \text{LM}_>(q_if_i)$.
+- If $h$ is nonzero, then $\text{LM}_>(h)$ is not divisible by any $\text{LM}_>(f_i)$.
+
+Each such expression is called a *weak standard representation* for $g$ with *quotients* $q_i$ and *remainder* $h$ (in terms of the $f_i$, with respect to $>$).
+If $g\in K[x])$, we speak of a *polynomial weak standard representation* if $u$ and the $q_i$ are elements of $K[x].$ While from a theoretical point of view it makes
+still sense to speak of fully reduced remainders, such remainders may not be computable (in finitely many steps).
+
+The OSCAR functions discussed below compute standard representations and polynomial weak standard representations, respectively.
+
+```@julia
+reduce(I::IdealGens, J::IdealGens; ordering::MonomialOrdering = default_ordering(base_ring(J)))
+```
+
+```@julia
+reduce(f::T, F::Vector{T}; ordering::MonomialOrdering = default_ordering(parent(f))) where {T <: MPolyElem}
+```
+
+```@julia
+reduce(F::Vector{T}, G::Vector{T}; ordering::MonomialOrdering = default_ordering(parent(F[1]))) where {T <: MPolyElem}
+```
+
+```@julia
+reduce_with_quotients(I::IdealGens, J::IdealGens; ordering::MonomialOrdering = default_ordering(base_ring(J)))
+```
+
+```@julia
+reduce_with_quotients_and_units(I::IdealGens, J::IdealGens; ordering::MonomialOrdering = default_ordering(base_ring(J)))
+```
+
+
+## Gröbner and Standard Bases
+
+Still keeping the notation introduced at the beginning of this section, let $I$ be an ideal of $K[x]_>$.
+Then the *leading ideal* of $I$ is the ideal of $K[x]$ defined by
+
+$\text{L}_>(I):=\langle \text{LT}_>(f) \mid f\in I\rangle\subset K[x].$
+
+A *standard basis* of $I$ is a finite set of elements of $I$ such that the leading terms of these elements
+generate $\text{L}_>(I)$. A finite subset of $K[x]_>$ is a *standard basis* if it is a standard basis for the
+ideal it generates.
+
+We call a standard basis $\{f_1,\dots, f_r\}\subset K[x]_>$ *minimal*  if  $\text{LM}_>(f_i)\neq \text{LM}_>(f_j)$,
+for $i\neq j$. We call it *reduced* if  it is minimal and no term of $f_i$ is divisible by $\text{LM}_>(f_j)$, for $i\neq j$.
+
+!!! note
+    The above definitions deviatate from the usual textbook definitions as we do not ask that the leading coefficients of the standard basis elements are 1.
+
+!!! note
+    The standard bases returned by OSCAR are always minimal in the sense above.
+
+If $>$ is global, that is, $K[x] = K[x]_>$, then we use the word *Gröbner basis* instead of standard basis.
```

Comment:
```suggestion
If $>$ is global, that is, $K[x] = K[x]_>$, then we use the expression *Gröbner basis* instead of standard basis.
```
(This is not a word, it's two words... ;-) )